core/src/main/java/greencity/controller/EventCommentController.java

```diff
@@ -59,34 +69,39 @@ public class EventCommentController {
         @ApiResponse(responseCode = "404", description = HttpStatuses.NOT_FOUND,
             content = @Content(examples = @ExampleObject(HttpStatuses.NOT_FOUND))),
     })
-    @PostMapping("/{eventId}")
-    public ResponseEntity<AddEventCommentDtoResponse> save(@PathVariable Long eventId,
+    @PostMapping
```

Comment:
Look, if I want save comment for event, I should go on /events/{eventId}/comments, because comment related to particular event. If i do this /events/comments/{eventId}, as you say, it turns out that event related to comment. That not RESTful.
I recommend you to read changelog file, for better understand my changes and refresh you knowledge about REST, using for example, this article: https://phauer.com/2015/restful-api-design-best-practices